forge-game/src/main/java/forge/game/staticability/StaticAbilityMustAttack.java

```diff
@@ -0,0 +1,44 @@
+package forge.game.staticability;
+
+import forge.game.Game;
+import forge.game.GameEntity;
+import forge.game.ability.AbilityUtils;
+import forge.game.card.Card;
+import forge.game.player.Player;
+import forge.game.zone.ZoneType;
+
+public class StaticAbilityMustAttack {
+
+    static String MODE = "MustAttack";
+
+    public static GameEntity mustAttackSpecific(final Card attacker) {
+        final Game game = attacker.getGame();
+        for (final Card ca : game.getCardsIn(ZoneType.STATIC_ABILITIES_SOURCE_ZONES)) {
+            for (final StaticAbility stAb : ca.getStaticAbilities()) {
+                if (!stAb.getParam("Mode").equals(MODE) || stAb.isSuppressed() || !stAb.checkConditions()) {
+                    continue;
+                }
+                if (stAb.matchesValid(attacker, stAb.getParam("Affected").split(","))) {
+                    if (stAb.hasParam("MustAttack")) {
+                        GameEntity e = AbilityUtils.getDefinedEntities(attacker, stAb.getParam("MustAttack"),
```

Comment:
you need to return all Entities here, not just the first match

it should probably be with a List in case some Requirements are for the same one

this matters because if creature has both "mustAttack A" and "mustAttack B" you are again free to choose either one because both would satisfy the maximum possible requirements